style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 10px;
+    background: #F5F5F5;
+    padding: 10px;
+}
+
+.gridContainer * {  /* applies to all the items in the grid */
+    background: #508EFF;
+    border-radius: 15px;
+    margin: 10px;
+}
+/* column control */
+.gridCol-2 {grid-template-columns: repeat(2, 1fr);}
```

Comment:
I think we'll only use the 2-column and 3-column versions, right? Let's just add those for now, it's easy enough to add the other ones later if we use them. But if, e.g. the 6-column one is there, it's easy for someone else to think that it's being used somewhere, when in fact it isn't.

Also, as a style thing, I'd prefer it if we always put newlines inside the braces, so:

```css
.gridCol-3 {
  grid-template-columns: repeat(3, 1fr);
}
```

instead of

```css
.gridCol-3 {grid-template-columns: repeat(3, 1fr);}
```